esinstall/src/index.ts

```diff
@@ -302,7 +303,8 @@ export async function install(
         const aliasEntry = findMatchingAliasEntry(installAlias, specifier);
         return aliasEntry && aliasEntry.type === 'package' ? aliasEntry.to : specifier;
       })
-      .sort(),
+      .map((specifier) => specifier.replace(/(\/|\\)+$/, '')) // remove trailing slash from end of specifier (easier for Node to resolve)
```

Comment:
This fixes a small but important problem: when using import maps, a `.` will resolve to `preact/` when using `path.join()`. This makes esinstall generate `preact/.js` rather than `preact.js`.

Here, if a slash is the very last character(s), we simply remove it. Shouldn’t affect Node generation, and it seems to produce more consistently-named files on esinstall’s end

Reply:
nice!